sal_interfaces/Dome/Dome_Events.xml

```diff
@@ -104,15 +104,15 @@
         <Description>Elevation axis (light/wind screen) target position and velocity.</Description>
         <item>
             <EFDB_Name>position</EFDB_Name>
-            <Description>Target position; nan if crawling.</Description>
-            <IDL_Type>int</IDL_Type>
+            <Description>Target position; nan for the crawlEl command.</Description>
+            <IDL_Type>float</IDL_Type>
             <Units>deg</Units>
             <Count>1</Count>
         </item>
         <item>
             <EFDB_Name>velocity</EFDB_Name>
             <Description>Target velocity.</Description>

```

Comment:
The velocity should be nan for the moveEl command since it will move to a position and then stop.

Reply:
Actually it should be 0 because the velocity will be known to be 0 (whereas target position truly makes no sense for the crawlEl command). I expanded the description.